.github/workflows/CI.yml

```diff
@@ -438,4 +438,38 @@ jobs:
         with:
           running_type: candidate
           opts: "-t=UploadCandidate -s=build/packages/.aoc-stack-test.yml"
-
+          
+  pr-soaking-test:
+    runs-on: ubuntu-latest
+    if: github.event_name == 'pull_request' && github.event.label.name == 'soaking'
```

Comment:
can we make label name to full "soaking test need" or something better? people can easy to understand from the PR?

Reply:
okay, sounds good! let me change it